tests/__main__.py

```diff
@@ -44,6 +48,9 @@ def convert_argv(argv):
             lint = True
             runtests = False
             break
+        elif arg.startswith('--junit-xml='):
+            junit_xml_file = arg[len('--junit-xml='):]
```

Comment:
This could also be spelled:

    _, _, junit_xml_file = arg.partition('=')

Reply:
Fanastic. I think `argparse` will remove this need, but I'll try and keep that in front of mind as I write further Python functions.